features/deploy.feature

```diff
@@ -40,6 +40,11 @@ Feature: Deploy
     Then the repo is cloned
     And the release is created
 
+  Scenario: REVISION and REVISION_TIME files are present
+    Given I make 1 deployment
+    And the REVISION file is created in the release
+    And the REVISION_TIME file is created in the release
```

Comment:
I think this would read more naturally:

```suggestion
    When I make 1 deployment
    Then the REVISION file is created in the release
    Then the REVISION_TIME file is created in the release
```